mos-platform/commodore/cbm.h

```diff
@@ -219,6 +219,7 @@ unsigned cbm_k_iobase (void) __attribute__((leaf));
 void cbm_k_listen (unsigned char dev);
 void *cbm_k_load(unsigned char flag, void *startaddr) __attribute__((leaf));
 unsigned char cbm_k_open (void) __attribute__((leaf));
+unsigned long cbm_k_rdtim (void);
```

Comment:
`__attribute__((leaf))`? `RDTIM` shouldn't have any callbacks outside the KERNAL, right?

Reply:
D'oh, of course, sorry.